src/operator/nn/mkldnn/mkldnn_base-inl.h

```diff
@@ -137,6 +137,10 @@ static inline bool SupportMKLDNN(const NDArray &input) {
       && SupportStorageMKLDNN(input.storage_type());
 }
 
+static inline bool MKLDNNEnvSet() {
+  return dmlc::GetEnv("USE_MKLDNN", true);
```

Comment:
Seems like you're setting the default to be `true`, would you please also add corresponding documentation for this env var on the webpage so that we're not causing surprise for users?